packages/tools/devtools/devtools-example/tsconfig.json

```diff
@@ -3,9 +3,10 @@
 	"compilerOptions": {
 		"rootDir": "src",
 		"outDir": "dist",
+		"module": "ESNext",
```

Comment:
Not opposed to this change, just curious what it is needed for?

Reply:
I probably can refer to @alexvy86's comments before. The test-until is not a lib as test lib. We should need to define it as ESM

Reply:
Ah, okay. Makes sense, and sounds good!